app/controllers/admin/comments_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Admin
+  class CommentsController < Admin::BaseController
+    load_and_authorize_resource
+
+    def index
+      @conferences_available = Conference.with_roles([:admin, :organizer, :cfp], current_user)
```

Comment:
@nTraum could you show us what you have tried so far so that we can guide you? Or elaborate on exactly what is giving you a hard time? Are you working on **Comment.accessible_by(current_ability)** or something else?